### DISCUSSION TITLE
Who uses tfcmt?

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [suzuki-shunsuke/tfcmt#1032] [Deprecated (General)]
Hi, I'd like to know who uses tfcmt.
And I'd like to add the section `Who uses tfcmt?` to the document.
If you or your organization uses tfcmt, please tell us!

Thank you for your cooperation!

https://github.com/suzuki-shunsuke/tfcmt#who-uses-tfcmt

## How to contribute

Please send pull requests to this repository to add your organization to the list.
If you don't want to send pull request, you can only post a comment to the discussion. Then we can send pull requests instead of you.

## Reference

- ArgoCD https://github.com/argoproj/argo-cd/blob/master/USERS.md
- tfaction https://github.com/suzuki-shunsuke/tfaction/discussions/1280

**Comment by uta8a** [comment]
Hi, I'm @uta8a from [Cybozu Inc](https://cybozu.co.jp/). We, "Cybozu Engineering Productivity Team" are using tfcmt to keep track of terraform plan, apply results on GitHub. Thanks for your great OSS!

I didn't see a section in the README, could you please put it as follows?

```md
- [Cybozu Inc / Engineering Productivity Team](https://cybozu.co.jp/)
```

We are very happy to have our name listed as a company that uses tfcmt!

**Reply by suzuki-shunsuke** [reply]
Hi @uta8a , thank you for your comment!
I'm really glad that tfcmt is used in your team!

I added the section to README.

https://github.com/suzuki-shunsuke/tfcmt#who-uses-tfcmt

Could you send a pull request?

**Reply by uta8a** [reply]
I sent a PR! https://github.com/suzuki-shunsuke/tfcmt/pull/1043
Thank you!

**Comment by akitok** [comment]
Hi! I'm @akitok from [CADDi Inc.](https://caddi.com/)
We use tfcmt to record the results of terraform apply in the PR comments.

I sent a [PR](https://github.com/suzuki-shunsuke/tfcmt/pull/1112).
Thank you so much!

**Comment by cubicdaiya** [comment]
Hi, I am @cubicdaiya from @sakura-internet. We use tfcmt to track results of terraform (plan|apply) in a pull-request.

Can you append it as follows?

```
- [SAKURA internet Inc.](https://www.sakura.ad.jp/)
```

Thank you for your good OSS!

**Reply by suzuki-shunsuke** [reply]
Hi @cubicdaiya ,
Thank you for your comment!
Could you send a pull request?

**Reply by cubicdaiya** [reply]
I sent the PR -> https://github.com/suzuki-shunsuke/tfcmt/pull/1158

Thank you.

**Comment by gion-pop** [comment]
Hi, I'm @gion-pop from [Unipos Inc.](https://www.unipos.co.jp/)
We use tfcmt to show and review terraform plan result in PR.

I sent a [PR](https://github.com/suzuki-shunsuke/tfcmt/pull/1163). Thank you for your work!
